R/test_wrapper.R

```diff
@@ -21,7 +21,14 @@ do_t.test <- function(df, value, key=NULL, ...){
   do_t.test_each <- function(df, ...){
     if(with_key){
       # use formula (`value_col`~`key_col`) for two sample t-test
-      model <- t.test(data=df, fml, ...)
+      model <- tryCatch({
+        t.test(data=df, fml, ...)
+      }, error = function(e){
+        if(e$message == "grouping factor must have exactly 2 levels"){
+          stop("grouping column has to have 2 unique values")
```

Comment:
Would it make sense to make grouping column title case, like Grouping Column so that it matches with what we show on command UI?